bin/shorebird.ps1

```diff
@@ -0,0 +1,95 @@
+# This is the windows equivalent of the `third_party/flutter/bin/internal/shared.sh` script
+# compiles `shorebird_cli/bin/shorebird_cli.dart` to `bin/cache/shorebird.shapshot`
+
+# We are running from $shorebirdRootDir\bin
+$shorebirdBinDir = (Get-Item $PSScriptRoot).FullName
```

Comment:
I have no idea what powershell style is, we should find a formatter: https://dscottraynsford.wordpress.com/2017/11/18/auto-formatting-powershell-in-visual-studio-code/

Reply:
There's one built into vs code that I've been using, but it doesn't seem to be very opinionated 🤷